backend/compact-connect/bin/generate_mock_data.py

```diff
@@ -0,0 +1,197 @@
+        'givenName': name_faker.first_name(),
+        'middleName': name_faker.first_name(),
+        'familyName': name_faker.last_name(),
+        # A few will have a suffix
+        'suffix': name_faker.suffix() if randint(0, 10) == 10 else None,
+        'homeAddressStreet1': faker.street_address(),
+        # Flip a coin, add secondary address line?
+        'homeAddressStreet2': faker.secondary_address() if choice([True, False]) else None,
+        'homeAddressCity': faker.city(),
+        # Some have email addresses, some don't
+        'emailAddress': faker.email() if choice([True, False]) else None,
+        'phoneNumber': f'+1{randint(1_000_000_000, 9_999_999_999)}'
+    }
+    license_data = _set_address_state(license_data, jurisdiction)
+    license_data = _set_dates(license_data)
+    return schema.dump(license_data)
+
+
+def _set_address_state(license_data: dict, jurisdiction: str) -> dict:
+    # 1/5 will have a military waiver
+    military = choice([False, False, False, False, True])
+    if military:
+        home_state = faker.state_abbr().lower()
+        license_data.update({
+            'homeAddressState': home_state,
+            'homeAddressPostalCode': faker.zipcode_in_state(state_abbr=home_state.upper()),
+            'militaryWaiver': military
+        })
+    else:
+        license_data.update({
+            'homeAddressState': jurisdiction,
+            'homeAddressPostalCode': faker.zipcode_in_state(state_abbr=jurisdiction.upper()),
+            # Explicitly set False for some, omit for others
+            'militaryWaiver': military if choice([True, False]) else None
+        })
+    return license_data
+
+
+def _set_dates(license_data: dict) -> dict:
+    date_of_birth = faker.date_of_birth()
+    # Issuance between when they were ~22 and ~40 years old, but still in the past
+    now = datetime.now(tz=UTC).date()
+    date_of_issuance = min(
+        date_of_birth + timedelta(days=randint(22 * 365, 40 * 365)),
+        now - timedelta(days=1)
+    )
+    # For simplicity, we'll assume that under-70-year-olds are active, over are inactive.
+    if date_of_birth + 70*timedelta(days=365) > now:
+        active = True
+        # We'll have renewal be within the last year, but on or after issuance.
+        date_of_renewal = max(
+            now - timedelta(days=randint(1, 365)),
+            date_of_issuance
+        )
+        # Expiry, one year from renewal
+        date_of_expiry = date_of_renewal + timedelta(days=365)
+    else:
+        active = False
+        # They renewed at some point in the last 20 years, but on or after their issuance date.
+        date_of_renewal = max(
+            date_of_issuance,
+            now - randint(1, 20)*timedelta(days=365)
+        )
+        # Their license expired a year after renewal, but no later than yesterday.
+        date_of_expiry = min(
+            date_of_renewal+timedelta(days=365),
+            now - timedelta(days=1)
```

Comment:
nit: spacing feels inconsistent around binary operators here. What's the style guide you use prescribe?

Reply:
Single space. Will fix.

Reply:
Is there a reason the python linter doesn't catch this?